lib/util/base64-js.js

```diff
@@ -0,0 +1,115 @@
+// from: https://github.com/beatgammit/base64-js/releases/tag/v1.2.0
+'use strict'
+
+exports.byteLength = byteLength
```

Comment:
Yup! Would be good to double check that it works for you, but it definitely worked for me.